### PR TITLE
Update seed to new jsonforms root component

### PR DESCRIPTION
**Pull request by eneufeld**
This updates the seed to use the new root component.
- Removed the store
- move necessary parameters into app.component

### CODE REVIEW

File: src/app/app.component.ts [hunk 1]
```diff
@@ -1,9 +1,68 @@
...
+  ];
+  uischema = uischemaAsset;
+  schema = schemaAsset;
+  data = dataAsset;
+  ajv = new AJV({
+    schemaId: 'auto',
+    allErrors: true,
+    jsonPointers: true,
+    errorDataPath: 'property'
+  });
```
Comment (sdirix) [inline src/app/app.component.ts:56]: I think we should use `createAjv` which is exported from JSON Forms as it automatically sets all options we need.

**Review by sdirix — DISMISSED** [review]
Works for me!

File: src/app/app.component.ts (hunk 1 above)
Comment (eneufeld) [reply src/app/app.component.ts]: good catch, I updated

**Review by sdirix — DISMISSED** [review]
Still works for me ;)

**Comment by sdirix** [comment]
Commit already included in https://github.com/eclipsesource/jsonforms-angular-seed/pull/10

**Pull request closed without merging**